### PR TITLE
update test page

### PR DESCRIPTION
**Pull request by Joffcom**
Adds widgets to the test page.

### CODE REVIEW

**Comment by StarfallProjects** [comment]
While you're at it could you add a test for the glossary? 
It should be like this: https://squidfunk.github.io/mkdocs-material/reference/tooltips/?h=glossary#adding-a-glossary

But Oleg has just encountered problems and submitted a fix: https://github.com/n8n-io/n8n-docs/pull/2223

I am really not sure why it would suddenly break, we've been using it since last year

File: docs-site-feature-tests/index.md [hunk 1]
```diff
@@ -52,3 +52,10 @@ Other images like this should expand on click:
 [This link should show a preview of the quickstart on hover](/try-it-out/quickstart/){ data-preview }
 
 [This link should NOT show a preview on hover](/try-it-out/quickstart/)
+
+## Embed Workflow
```
Comment (StarfallProjects) [inline docs-site-feature-tests/index.md:56]:
```suggestion
## Embed Workflow

You should see an interactive workflow, similar to the templates pages or forum.
```

File: docs-site-feature-tests/index.md [hunk 2]
```diff
@@ -52,3 +52,10 @@ Other images like this should expand on click:
...
+[[ workflowDemo("https://api.n8n.io/workflows/templates/655") ]]
+
+## Templates Widget
```
Comment (StarfallProjects) [inline docs-site-feature-tests/index.md:59]:
```suggestion
## Templates Widget

You should see a list of three templates, followed by links to the integrations page on the website, and the templates search.
```

File: docs-site-feature-tests/index.md [hunk 3]
```diff
@@ -52,3 +52,10 @@ Other images like this should expand on click:
...
+<!-- see https://www.notion.so/n8n/Pull-in-templates-for-the-integrations-pages-37c716837b804d30a33b47475f6e3780 -->
+[[ templatesWidget('Test Title', 'http-request') ]]
```
Comment (StarfallProjects) [inline docs-site-feature-tests/index.md:61]: This won't work. The title of the page is used to create the API call to load the template. You need to provide an actual title if you want it to load templates.

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
See comments

File: docs-site-feature-tests/index.md (hunk 3 above)
Comment (Joffcom) [reply docs-site-feature-tests/index.md]: Changed

**Pull request merged**